### DISCUSSION TITLE
Task Switcher will work with labwc?

### DISCUSSION THREAD

**Discussion by ffs-akl** [LBCrion/sfwbar#248] [Q&A] [answered]
in my case the switcher didn't work with labwc.  I'm wondering if the config file or labwc is causing this?

**Comment by LBCrion** [comment, marked as answer]
How are you invoking the switcher? You need to bind a key in labwc to send
SIGUSR1 to sfwbar to trigger it.

**Reply by ffs-akl** [reply]
Thanks  It worked.
